DEVEL.md

```diff
@@ -181,34 +181,92 @@ container images with the executables under test must be available in the
 cluster. If these two conditions are satisfied, run the tests with:
 
 ```bash
-$ go test -v ./test/e2e/...
+# Run all e2e tests in this repository
+go test -v ./test/e2e/...
 ```
 
-In case you want to run only certain tests, e.g., QAT ones, run:
+If you need to specify paths to your custom `kubeconfig` containing
+embedded authentication info then add the `-kubeconfig` argument:
 
 ```bash
-$ go test -v ./test/e2e/... -args -ginkgo.focus "QAT"
+go test -v ./test/e2e/... -args -kubeconfig /path/to/kubeconfig
 ```
 
-If you need to specify paths to your custom `kubeconfig` containing
-embedded authentication info then add the `-kubeconfig` argument:
+The full list of available options can be obtained with:
 
 ```bash
-$ go test -v ./test/e2e/... -args -kubeconfig /path/to/kubeconfig
+go test ./test/e2e/... -args -help
 ```
 
-The full list of available options can be obtained with:
+In most cases, it would not be possible to run all E2E tests in one system.
+For running a subset of tests, there are labels that you can use to pick out specific parts.
+You can run the tests with:
+```bash
+# Run a subset of tests
+go test -v ./test/e2e/... -args -ginkgo.focus <some labels> -ginkgo.skip <some labels>
+```
+
+#### Table of Labels
+
+| Device | Mode             | Resource    | App                            |
+|:-------|:-----------------|:------------|:-------------------------------|
+| `dlb`  |-                 | `pf`, `vf`  | `libdlb`                       |
+| `dsa`  |-                 | `dedicated` | `accel-config`                 |
+| `fpga` | `af`, `region`   |             | `opae-nlb-demo`                |
+| `gpu`  |-                 | `i915`      | `busybox`, `tensorflow`        |
+| `iaa`  |-                 | `dedicated` | `accel-config`                 |
+| `qat4` | `dpdk`           | `dc`        | `openssl`                      |
+| `qat4` | `dpdk`           | `cy`        | `openssl`, `crypto-perf`       |
+| `qat2` | `dpdk`           | `generic`   | `crypto-perf`, `compress-perf` |
+| `qat2` | `kernel`         | `cy1_dc0`   | `busybox`                      |
```

Comment:
I wonder if it's possible to ask these combinations from go test?

Reply:
what combinations do you mean?
combinations of qat4 and qat2?

Reply:
to have this table printed out automatically by `go test` rather than having to maintain the table here

Reply:
yep, as the Apps and other tags are included in the test descriptions, it would be nice if this sort of table could be constructed programmatically to leave out human error.

Reply:
umm, is there a way that you know? Sorry, go test is still a bit tricky to me..
Is it ok if we implement it later?

Reply:
Don't know. It was more of a "I wonder if" thing. It's fine to implement later.